src/symbolic/HybridTimedAutomata/TimedHybridAutomata.jl

```diff
@@ -0,0 +1,645 @@
+export TimedHybridAutomata
+
+module TimedHybridAutomata
+
+using HybridSystems, MathematicalSystems, StaticArrays, JuMP
```

Comment:
Prefer using `import` inside the `src` module. `using` is fine in the test and docs but in the code, stick to import

Reply:
Cfr https://jump.dev/JuMP.jl/stable/developers/style/#using-vs.-import

Reply:
By the way, you can probably replace `using JuMP` by `import MathOptInterface as MOI`